.github/workflows/static_checks.yml

```diff
@@ -128,3 +128,29 @@ jobs:
         if: always()
         run: |
           bazel run //scripts:kdoc_validity_check -- $(pwd) scripts/assets/kdoc_validity_exemptions.pb
+
+  third_party_dependencies_check:
+    name: Maven Dependencies Checks
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Bazel
+        uses: abhinavsingh/setup-bazel@v3
+        with:
+          version: 4.0.0
+
+      - name: Maven Repin Check
+        if: always()
+        run: |
+          REPIN=1 bazel run @unpinned_maven//:pin
+
+      - name: License Details Check
+        if: always()
+        run: |
+          bazel run //scripts:generate_maven_dependencies_list -- $(pwd) third_party/maven_install.json scripts/assets/maven_dependencies.textproto scripts/assets/maven_dependencies.pb
```

Comment:
I have added this to cover the cases - when the developers re-run the script to update maven_dependencies.textproto but forget to complete license details of some new dependencies, this check will fail if some dependency's license details are not completed.
WDYT @BenHenning?